client/components/Itinerary.js

```diff
@@ -168,6 +141,7 @@ const Itinerary = ({ currentUser, currentTrip, day }) => {
           setActivities(newActivities);
         }}
       />
+      <br />
```

Comment:
use css to format spacing in components, not br tags

Reply:
fixed